build-local.py

```diff
@@ -0,0 +1,228 @@
+                f.write('<li class="version-menu"><a href="/{lang}/{lang}-{tag}">{lang}-{tag}</a></li>\n'.format(lang=lang, tag=tag))
+            else:
+                f.write('<li class="version-menu"><a href="/{lang}">{lang}</a></li>\n'.format(lang=lang))
+
+
+    # copy over the full version-menu listing to datajoint_theme FIRST,
+    # then build individual folders, and copy to loc_built_site folder
+    for folder in to_make:
+        shutil.copy2(path.join('datajoint_theme', 'version-menu.html'), path.join(folder, "datajoint_theme", "version-menu.html"))
+        if platform.system() == "Windows":
+            subprocess.Popen(["sphinx-build", "-b", "html", ".", "..\_build\html"], cwd=path.join(folder, "contents")).wait()  
+            subprocess.Popen(["sphinx-build", "-b", "latex", ".", "..\_build\latex"], cwd=path.join(folder, "contents")).wait()
+            if path.exists(path.join(folder,"site")):
+                shutil.rmtree(path.join(folder, "site"))
+            os.makedirs(path.join(folder, "site"))
+            copy_contents(path.join(folder, "_build", "html"), path.join(folder, "site"))
+        else:
+            subprocess.Popen(["make", "site"], cwd=folder).wait()
+
+        try:
+            subprocess.Popen(["pdflatex", "DataJointDocs.tex"], cwd=path.join(folder, '_build', 'latex')).wait()
+            subprocess.Popen(["pdflatex", "DataJointDocs.tex"], cwd=path.join(folder, '_build', 'latex')).wait()
+        except:
+            print("Latex environment not set up - no pdf will be generated")
+
+        # 'matlab' from `build-local/matlab/contents/...` or 'matlab-v3.2.4' from `build-local/matlab-v3.2.4/contents/...`
+        lang_version = folder.split(os.sep)[1]
+        lang = lang_version[:6] # workaround to make sure /matlab or /python exists for the pre-set dj_root_theme
+        # dumping all contents of site/ inside something like 'loc_built_site/python/python-v0.10.0'
+        shutil.copytree(path.join(folder, "site"), path.join('loc_built_site', lang, lang_version))        
+
+        if path.exists(path.join(folder, '_build', 'latex', 'DataJointDocs.pdf')):
+            os.rename(path.join(folder, '_build', 'latex', 'DataJointDocs.pdf'), path.join(folder, '_build', 'latex', 'DataJointDocs_{}.pdf'.format(lang_version)))
+            shutil.copy2(path.join(folder, '_build', 'latex', 'DataJointDocs_{}.pdf'.format(lang_version)), path.join('loc_built_site', lang, lang_version))
+
+        copy_contents('dj_root_theme', 'loc_built_site')
+        copy_contents(path.join('loc_built_site', lang, lang_version), path.join('loc_built_site', lang))
+        copy_contents(path.join('loc_built_site', 'python', '_static'), path.join('loc_built_site', '_static'))
+
+
+##########################################################
+####====== begin building local version doc here ======####
+
+# if build_config file exists, override the default git_url and/or local_lang_path values with config values
+try:
```

Comment:
This block should already be under `if __name__ == "__main__":` - it doesn't make sense to execute these lines unless the whole thing is run as a script.